p2p/src/network/identify/p2p_network_identify_state.rs

```diff
@@ -39,4 +39,8 @@ impl P2pNetworkIdentifyState {
             .get_mut(peer_id)
             .map_or(false, |m| m.remove(stream_id).is_some())
     }
+
+    pub fn remove_peer(&mut self, peer_id: &PeerId) {
```

Comment:
Let's give it a name that better describes its functionality.

Reply:
Would `purge_peer_state` do?